shared-data/default-theme/html/jsapi/global/eventlog.js

```diff
@@ -63,11 +63,13 @@ EventLog.poll = function() {
   //
   //time for server to wait for new events
   var waittime = 30;
+  //extra time to account for processing time on backend
+  var buffer = 5;
```

Comment:
Variable names descriptive, LGTM. See https://github.com/mailpile/Mailpile/issues/2168#issuecomment-442823050 for 5 sec buffer reasoning